assets/dev/js/editor/controls/media.js

```diff
@@ -121,12 +132,38 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 			return;
 		}
 
-		const currentControlValue = this.getControlValue();
+		const currentControlValue = this.getControlValue(),
+			placeholder = this.getControlPlaceholder();
+
+		const isSelectedImage = ( '' !== currentControlValue?.id ),
+			isSelectedPlaceholder = placeholder?.id;
+
+		if ( ! isSelectedImage && ! isSelectedPlaceholder ) {
+			return;
+		}
+
+		if ( ! isSelectedImage && isSelectedPlaceholder ) {
```

Comment:
Can you extract the condition to a meaningful variable?

Reply:
Like what?